src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+function toNumber(salary) {
+  return parseFloat(salary.substring(1).replaceAll(',', ''));
+}
+
+function sortList(list) {
+  const newList = document.querySelector(list);
+
+  [...newList.children]
+    .sort((a, b) => toNumber(b.dataset.salary) - toNumber(a.dataset.salary))
+    .forEach(node => newList.appendChild(node));
+}
+
+function getEmployees(list) {
+  const newList = document.querySelector(list);
```

Comment:
```suggestion
function getEmployees() {
  const list = document.querySelector('ul');
```
я б навіть винесла `const list = document.querySelector('ul'); ` повище в змінну, щоб можна було в обох функціях перевикористати)